go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/jhump/protoreflect
 
+go 1.12
+
 require (
-	github.com/golang/protobuf v1.1.0
-	golang.org/x/net v0.0.0-20180530234432-1e491301e022
-	golang.org/x/text v0.3.0 // indirect
```

Comment:
This is correctly removed - these are the direct dependencies we need to have specified.